<?xml version="1.0" ?>
<launch>
  <node pkg="medibotv4" type="odom_tf.py" name="odom_tf">
    <param name="base_width" value = "0.498"/>
    <param name="ticks_meter" value = "89"/>
    <param name="publish_tf" value = "false"/>
    <param name="odom_frame_id" value = "odom"/>
    <param name="base_frame_id" value = "base_link"/> 
  </node>
</launch>

